static/js/portico/desktop-login.js

```diff
@@ -0,0 +1,14 @@
+const { key, check } = window.electron_bridge.decrypt_clipboard();
```

Comment:
This probably should have a block comment explaining the API between this and the desktop app code.

Overall I think how this model only has the ~30 lines of code on the desktop app side.